files/image_config/monit/memory_checker

```diff
@@ -140,7 +140,12 @@ def get_running_container_names():
         running_container_list = docker_client.containers.list(filters={"status": "running"})
         running_container_names = [ container.name for container in running_container_list ]
     except (docker.errors.APIError, docker.errors.DockerException) as err:
-        syslog.syslog(syslog.LOG_ERR,
+        if "('Connection aborted.', FileNotFoundError(2, 'No such file or directory'))" in str(err):
```

Comment:
@vadymhlushko-mlnx IMHO, better to switch to `systemd` direct calls or use `pystemd`

Reply:
Alinged